win32ss/user/ntuser/callback.c

```diff
@@ -980,6 +1007,61 @@ co_IntClientThreadSetup(VOID)
 
    return Status;
 }
+ 
+HANDLE FASTCALL
+co_IntLoadImage(LPCWSTR name, UINT type, INT desiredx, INT desiredy, UINT flags)
+{
+   HANDLE Handle;
+   NTSTATUS Status;
+   ULONG ArgumentLength, ResultLength;
+   PVOID Argument, ResultPointer;
```

Comment:
`Argument` is not needed, you can use `Common` directly in `KeUserModeCallback` without any casting.

Use can see an example here: https://github.com/reactos/reactos/blob/be970d8d313e9af71990c8b6025e8fcfa2f0c1b6/win32ss/user/rtl/text.c#L971

Reply:
There are cases where the capture buffer is needed, though.

Reply:
OK, so I shouldn't use this suggestion, right?